src/translations/en.json

```diff
@@ -926,6 +926,10 @@
           },
           "learn_more": "Learn more"
         },
+        "updates": {
+          "version_available": "You have {version}, {version_latest} is available",
```

Comment:
Those two versions only separated by a comma might look slightly weird. Perhaps adding a filler word in between such as "but" or "however" or alternatively "newer", might make it easier to read.

Reply:
We could even consider skipping the old version.